Interfaces/API/Dirac.py

```diff
@@ -63,7 +63,7 @@
 def parseArguments(args):
   argList = []
   for arg in args:
-    argList += arg.split(',')
+    argList += [a.strip() for a in arg.split(',') if a]
```

Comment:
Just in case someone manages to accidentally pass a variable containing only whitespace?
```suggestion
    argList += [a.strip() for a in arg.split(',') if a.strip()]
```

Reply:
even better